src/frontend/magnify/src/controller/default.controller.ts

```diff
@@ -0,0 +1,26 @@
+import Controller from './interface';
+import axios, { AxiosInstance } from 'axios';
+
+interface IDefaultControllerOptions {
```

Comment:
Is there a mispell?
```suggestion
interface DefaultControllerOptions {
```

Reply:
Indeed. I'm used to name my interfaces beginning by a I, but we didn't follow this rule in this project, so I remove it.